package.json

```diff
@@ -138,33 +146,94 @@
     "web components"
   ],
   "scripts": {
-    "build": "npm-run-all build:sass sass:render",
-    "build:test": "npm-run-all test linters",
-    "build:release": "npm-run-all build build:test build:api build:docs bundler types postinstall",
-    "build:ci": "npm-run-all sweep build:release",
-    "build:api": "wca analyze 'src/auro-form.js' --outFiles docs/api.md",
-    "build:dev:assets": "npm-run-all build:sass:component postCss:component sass:render build:docs",
-    "build:docs": "node scripts/generateDocs.mjs",
-    "build:sass": "npm-run-all build:sass:component postCss:component sass:render",
-    "build:sass:component": "sass --no-source-map src:src",
-    "build:watch": "nodemon -e scss,js,html --watch src --watch apiExamples/**/* --exec npm run build:dev:assets",
+   "dev": "npm run dev:input",
+   "build": "npm-run-all build:input build:dropdown build:sass sass:render types",
+   "build:test": "npm-run-all test linters",
+   "build:release": "npm-run-all build build:test build:api build:docs bundler postinstall",
+   "build:ci": "npm-run-all sweep build:release",
+   "postinstall": "node ./node_modules/@aurodesignsystem/auro-library/scripts/build/postinstall.mjs",
+   "types": "tsc",
+
+   "build:api": "wca analyze src/auro-dropdown.js --outFiles docs/api.md",
+   "build:api:input": "wca analyze \"./components/input/src/*.js\" --outFiles docs/api.md",
+   "build:api:dropdown": "wca analyze \"./components/dropdown/src/*.js\" --outFiles docs/api.md",
+   
+   "build:input": "npm-run-all build:version:input build:sass:input sass:render:input types",
+   "build:dropdown": "npm-run-all build:version:dropdown build:sass:dropdown sass:render:dropdown types",
+   
+   "dev:input": "concurrently --kill-others \"npm run build:watch:input\" \"npm run serve:input\"",
+   "dev:dropdown": "concurrently --kill-others \"npm run build:watch:dropdown\" \"npm run serve:dropdown\"",
+   
+   "build:demo": "npm-run-all build demo:rm:build demo:new:build demo:copy:index:input demo:copy:demo:input demo:update:index",
+   "build:demo:input": "npm-run-all build:input demo:rm:build demo:new:build demo:copy:index:input demo:copy:demo:input demo:update:index",
+   "build:demo:dropdown": "npm-run-all build:dropdown demo:rm:build demo:new:build demo:copy:index:dropdown demo:copy:demo:dropdown demo:update:index",
+   
+   "demo:copy:demo:input": "copyfiles -u 1 -V \"./components/input/src/demo/**/*\" ./build/demo",
```

Comment:
See comments from input and dropdown`s package.json about updating script names. If this is too much to handle now, create a ticket about refactoring the scripts to match the more modernized way of writing them that we see in most of the other repos.

Reply:
Updates to `package.json` are intended for this ticket: https://github.com/AlaskaAirlines/auro-form/issues/12

I will add these comments there to track the work.